PocketKit/Sources/Sync/Requests.swift

```diff
@@ -7,12 +7,20 @@ import CoreData
 
 public enum Requests {
     public static func fetchItems() -> NSFetchRequest<Item> {
+        let request = fetchAllItems()
```

Comment:
I think the word `all` may be overloaded. `All` items may be interpreted as all items a user has, including unarchived, archived, and deleted, but here, `all` means "unarchived and not deleted". What do you think about renaming this? `fetchActiveItems`, `fetchCurrentItems`, `fetchListItems`, `fetchSomeOtherWordItems`?

Reply:
well... `All` here actually means "anything in the table". the archived/deleted flags only come into account in `fetchItems` request (which adds predicates to `fetchAllItems`)

Reply:
Ah! You're right. My interpretation of this was incorrect :)